q2cli/core/config.py

```diff
@@ -0,0 +1,100 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2019, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import configparser
+
+import click
+
+import q2cli.util
+
+
+class cli_config():
+    path = os.path.join(q2cli.util.get_app_dir(), 'cli-colors.theme')
+    valid_types = ['options', 'type', 'default_arg', 'command', 'emphasis',
+                   'problem', 'errors', 'required', 'success']
+    valid_stylings = ['fg', 'bg', 'bold', 'dim', 'underline', 'blink',
```

Comment:
I think you have enough of these that a set-literal may be better than a list-literal, since your operations are mostly `in`.